prow/gerrit/adapter/adapter_test.go

```diff
@@ -295,27 +296,33 @@ func TestHandleInRepoConfigError(t *testing.T) {
 			startingFailures: map[string]bool{changeHash: true},
 			expectedFailures: map[string]bool{changeHash: true},
 		},
-		{
-			name:             "Resolved error sends error again, resend review",
-			err:              errors.New("InRepoConfigError"),
-			expectedReview:   true,
-			startingFailures: map[string]bool{},
-			expectedFailures: map[string]bool{changeHash: true},
-		},
```

Comment:
This test was functionally identical of the "First time error send review" so I removed it.